### DISCUSSION TITLE
Commandpost with Premiere Pro not working

### DISCUSSION THREAD

**Discussion by bimmeracer** [CommandPost/CommandPost#3339] [Technical Support]
Once I install the software and program the keys for Premiere Pro, do I need to do anything else to get Resolve Speed Editor to work with Premiere Pro? 

When I use the Resolve Speed Editor, nothing happens in PP. It works fine in Davinci Resolve.

**Comment by latenitefilms** [comment]
Can you share a screenshot of your Speed Editor layout for DaVinci Resolve?

**Reply by strykedits** [reply]
Here is my latest video on the topic:   https://www.youtube.com/watch?v=PcyfasCqSJU
